dpnp/dpnp_iface_arraycreation.py

```diff
@@ -1082,6 +1083,37 @@ def ones_like(x1, dtype=None, order='C', subok=False, shape=None):
     return call_origin(numpy.ones_like, x1, dtype, order, subok, shape)
 
 
+def ptp(arr, axis=None, out=None, keepdims=numpy._NoValue):
+    """
+    Range of values (maximum - minimum) along an axis.
+
+    For full documentation refer to :obj:`numpy.ptp`.
+
+    Limitations
+    -----------
+    Input array is supported as :obj:`dpnp.ndarray`.
+    Parameters ``out`` and ``keepdims`` are supported only with default values.
+    """
+    arr_desc = dpnp.get_dpnp_descriptor(arr)
+    if arr_desc:
+        if axis is not None and not isinstance(axis, int):
+            pass
```

Comment:
You could decrease indentations with the following way:
```suggestion
    if not arr_desc:
        pass
    elif axis is not None and not isinstance(axis, int):
        pass
```